perceval/utils/logging/__init__.py

```diff
@@ -47,6 +47,8 @@ def _my_excepthook(excType, excValue, this_traceback):
                  exc_info=(excType, excValue, this_traceback))
 
 
+sys.excepthook = _my_excepthook
```

Comment:
This is already done in use_python_logger or use_perceval_logger, either remove it from here or remove the call in both methods

Reply:
It wasn't working at Perceval import (now it does)